include/outputs

```diff
@@ -18,41 +18,22 @@ textPass(){
   fi
 
   PASS_COUNTER=$((PASS_COUNTER+1))
-  if [[ "$MODE" == "csv" ]]; then
+  if [[ "$MODE" == "csv" || "$MODE" == "json" || "$MODE" == "json-asff" || "$MODE" == "securityhub" ]]; then
     if [[ $2 ]]; then
       REPREGION=$2
     else
       REPREGION=$REGION
     fi
-    echo "$PROFILE${SEP}$ACCOUNT_NUM${SEP}$REPREGION${SEP}$TITLE_ID${SEP}PASS${SEP}$ITEM_SCORED${SEP}$ITEM_LEVEL${SEP}$TITLE_TEXT${SEP}$1"
-  elif [[ "$MODE" == "json" ]]; then
-    if [[ $2 ]]; then
-      REPREGION=$2
-    else
-      REPREGION=$REGION
+    if [[ "$MODE" == "csv" ]]; then
+      echo "$PROFILE${SEP}$ACCOUNT_NUM${SEP}$REPREGION${SEP}$TITLE_ID${SEP}PASS${SEP}$ITEM_SCORED${SEP}$ITEM_LEVEL${SEP}$TITLE_TEXT${SEP}$1"
+    elif [[ "$MODE" == "json" ]]; then
+      generateJsonOutput "$1" "Pass"
+    elif [[ "$MODE" == "json-asff" ]]; then
+      generateJsonAsffOutput "$1" "PASSED" "INFORMATIONAL"
+    elif [[ "$MODE" == "securityhub" ]]; then
```

Comment:
We no longer need this here and for the integration we can use a dedicated function in folder `prowler/include/` file called `securityhub_integration`. 

For the function it self we have to make sure next:
- Region, remember that Prowler gets region from the config file initially or whatever that you specify with `-r`.
- If SH is enabled in that region. 
- For the command we have to add next variables:
`$AWSCLI securityhub --region $REGION $PROFILEOPTS \
batch-import-findings \
--findings "$(generateJsonAsffOutput "$1" "PASSED" "INFORMATIONAL")" | jq -M -r 'if .SuccessCount == 1 then "Successfully submitted finding" else "Failed to upload finding" end'`
- I would move that if outside the `jq`

Reply:
These points should all be addressed now